lib/timers.js

```diff
@@ -283,7 +283,10 @@ function listOnTimeout(list, now) {
     // Check if this loop iteration is too early for the next timer.
     // This happens if there are more timers scheduled for later in the list.
     if (diff < msecs) {
+      const prev = list.expiry;
```

Comment:
I think this can all be replaced by `list.expiry = Math.max(timer._idleStart + msecs, now + 1);`

Reply:
Thanks for the review! 
I'll fix it 😄